package.json

```diff
@@ -28,5 +30,8 @@
   "engines": {
     "node": "^16.11.0 || ^18.0.0",
     "npm": "^8.0.0 || ^9.0.0"
+  },
+  "dependencies": {
+    "node-fetch": "^2.6.9"
```

Comment:
Might make sense to create a new package.json in `e2e` to keep its dependencies separate. Including `node-fetch` here makes it available to all the regular packages, which is a potential subtle foot gun.

Reply:
Makes sense but we'll need to do this in a follow-up ticket as well because it'll need a bunch of other changes like updating `bump-versions` script to keep the version on this new package.json file in sync with the rest of the monorepo and have it in sync with package-lock.json to state some.